sealtk/gui/Player.cpp

```diff
@@ -504,7 +594,82 @@ void PlayerPrivate::updateDetectedObjectVertexBuffers()
 }
 
 //-----------------------------------------------------------------------------
-void PlayerPrivate::drawImage(QOpenGLFunctions* functions)
+void PlayerPrivate::computeLevels(LevelsPair const& temporaryLevels)
+{
+  QTE_Q();
+
+  auto const t = this->timeStamp.get_time_usec();
+
+  // Set up task to compute percentile levels
+  auto const pd = this->percentileDeviance;
+  auto const pt = this->percentileTolerance;
+  auto* const task = new core::AutoLevelsTask{this->image, pd, pt};
+
+  // Hook up receipt of results from task
+  QObject::connect(
+    task, &core::AutoLevelsTask::levelsUpdated,
+    q, [t, cookie=this->percentileCookie, this](float low, float high){
```

Comment:
GCC 5.4.0 says:

`warning: lambda capture initializers only available with -std=c++14 or -std=gnu++14`

Reply:
Oh... it looks like this depends on CMake changes from #21.

Reply:
Right... I just merged in the relevant bits.